src/main/java/com/codedx/burp/ContextMenuFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2016 Code Dx, Inc. - http://www.codedx.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ package com.codedx.burp;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.swing.JMenuItem;
+import javax.swing.JOptionPane;
+
+import org.apache.http.NameValuePair;
+
+import burp.BurpExtender;
+import burp.IBurpExtenderCallbacks;
+import burp.IContextMenuFactory;
+import burp.IContextMenuInvocation;
+import burp.IScanIssue;
+
+public class ContextMenuFactory implements IContextMenuFactory{
+
+	private BurpExtender burpExtender;
+	private IBurpExtenderCallbacks callbacks;
+	
+	public ContextMenuFactory(BurpExtender be, IBurpExtenderCallbacks cb){
+		burpExtender = be;
+		callbacks = cb;
+	}
```

Comment:
I find that a good convention for constructor parameters is to name them exactly what the field is called, then use `this.thing = this`. I'd use that here, e.g.
```java
public ContextMenuFactory(BurpExtender be, IBurpExtenderCallbacks callbacks){
  this.burpExtender = burpExtender;
  this.callbacks = callbacks;
}
```